modules/dynatrace-activegate/providers.tf

```diff
@@ -0,0 +1,5 @@
+provider "azurerm" {
```

Comment:
never add a provider in a module, inject it in as configuration and as part of `required_providers` block.

It's a nightmare updating/removing/changing code that has had providers in modules, can go into more detail if needed but otherwise just please move it out 😄

Reply:
yeah was thinking that, was just running a test - will move out shortly

Reply:
Will give you a shout when its ready for review

Reply:
Resolved now